game-core/src/main/java/games/strategy/triplea/image/ImageColorizer.java

```diff
@@ -0,0 +1,93 @@
+package games.strategy.triplea.image;
+
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+
+/**
+ * Class used to colorize images. Based on finding the luminance from the original image then
+ * applying hue, saturation, and brightness to colorize it.
+ */
+public class ImageColorizer {
+
+  private static final int MAX_COLOR = 256;
+  private static final float LUMINANCE_RED = 0.2126f;
```

Comment:
Are there any resources or references to give background information on these luminosity constants?

Reply:
Added more comments to class referencing luminosity.